sources/Context.cpp

```diff
@@ -327,6 +397,8 @@ Context::exportOutputs() {
 
   // create output directory
   file::path outputDir(config_.outputDir());
+  outputs::Job::setStartAndDuration(config_.getStartTime(),
```

Comment:
We should review the definition of start time and duration of simulation in `outputs::Job `class: now they won't be constants, but configurable attributes. They should not be declared static in `Job.h`.

Additionally, for contingency analysis now we will have multiple `Job` instances. Having all the private attributes declared static could lead us into errors later. I think it would be better to have proper objects instead of a `Job` singleton.

Reply:
We could leave this change for later. Let's first close a version where we use the Job as it is, a singleton. Just let the start time and duration be changed from the configuration.